model/src/lib.rs

```diff
@@ -75,6 +75,7 @@ pub mod guild;
 pub mod id;
 pub mod invite;
 pub mod oauth;
+pub mod scheduled_event;
```

Comment:
Could this not be under `guild` directly? (`model::guild::GuildScheduledEvent` (re-exported) and `model::guild::scheduled_event::{EntityMetadata, EntityType, PrivacyLevel, Status}`)

Reply:
I matched the Resources tab on the docs: 
<img width="261" alt="image" src="https://user-images.githubusercontent.com/1760581/147806466-d4a4d095-21c8-40aa-9cee-7c78cc216e5d.png">